foundation/eclipselink.core.test/src/org/eclipse/persistence/testing/framework/UniversalSessionTestAdapter.java

```diff
@@ -21,9 +21,9 @@
  *  on any session configuration.
  *  @author  Stephen McRitchie
  *  @since   10 used for testing flashback queries on different sessions.
- *  @see org.eclipse.persistence.testing.ClientServerTests.ClientSessionTestAdapter
- *  @see org.eclipse.persistence.testing.tests.sessionbroker.ClientSessionBrokerTestAdapter
- *  @see org.eclipse.persistence.testing.tests.flashback.HistoricalSessionTest
+ *  @see "org.eclipse.persistence.testing.ClientServerTests.ClientSessionTestAdapter"
```

Comment:
change to "org.eclipse.persistence.testing.tests.clientserver.ClientSessionTestAdapter"